tests/test_pipelines.py

```diff
@@ -917,7 +917,7 @@ def test_stable_diffusion_fast_ddim(self):
         image_slice = image[0, -3:, -3:, -1]
 
         assert image.shape == (1, 512, 512, 3)
-        expected_slice = np.array([0.8354, 0.83, 0.866, 0.838, 0.8315, 0.867, 0.836, 0.8584, 0.869])
```

Comment:
Any idea why this was changed (maybe I need to double check here real quick with SD github)

Reply:
Could be all the way back from the generator's cpu->gpu move, so please check if you have the script neraby!

Reply:
Checking now!

Reply:
Results on master are 1-to-1 the same as original CompVis repo still (checked with scripts from: https://github.com/huggingface/diffusers/pull/182) -> so change is good to go!